tutorials/create-a-usb-stick-on-ubuntu/create-a-usb-stick-on-ubuntu.md

```diff
@@ -1,60 +1,86 @@
 tags: tutorial,installation,usb,ubuntu,desktop
 difficulty: 2
 status: Published
-published: 2017-05-31
+published: 2017-07-11
+author: Canonical Web Team <webteam@canonical.com>
 
 ---
 
 # Create a bootable USB stick on Ubuntu
 
-## Getting started
-Duration: 0:05
+## Overview
+Duration: 0:01
+
+With a bootable Ubuntu USB stick, you can:
 
-To create a USB stick from which you can install Ubuntu, you must first [download Ubuntu](https://www.ubuntu.com/download) and, if you want, [verify the download](/tutorial/tutorial-how-to-verify-ubuntu).
+- Install or upgrade Ubuntu.
+- Test out the Ubuntu desktop experience without touching your PC configuration.
+- Boot into Ubuntu on a borrowed machine or from an internet cafe.
+- Use tools installed by default on the USB stick to repair or fix a broken configuration.
 
-Then, follow these instructions:
+Creating a bootable Ubuntu USB stick is very simple, especially from Ubuntu itself, and we're going to cover the process in the next few steps.
 
-1. Insert a USB stick with at least 2GB of free space
-2. Open the dash and search for Startup Disk Creator
-3. Select the Startup Disk Creator to launch the app
+Alternatively, we also have tutorials to help you create a bootable USB stick from both [Microsoft Windows](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-windows) and [Apple macOS](https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-macos).
 
-## Find the Startup Disk Creator
+![image](https://assets.ubuntu.com/v1/0e7183ed-laptop-ubuntu.png)
+
+## Requirements
 Duration: 0:01
 
-Find and start the Startup Disk Creator app.
+You will need:
 
-![screenshot](https://assets.ubuntu.com/v1/be194ecf-usb-1604-startup-disk-creator.png?w=550)
+1. A 2GB or larger USB stick/flash drive. 
+1. Ubuntu Desktop 14.04 or later installed.
+1. An Ubuntu ISO file. See [Get Ubuntu](https://www.ubuntu.com/download) for download links.
 
-## Choose the ISO
-Duration: 0:01
+![screenshot](https://assets.ubuntu.com/v1/bfb2d510-ubuntu-iso-download.png)
+
+## Launch Startup Disk Creator
+
+We're going to use an application called 'Startup Disk Creator' to write the ISO image to your USB stick. This is installed by default on Ubuntu, and can be launched as follows:
+
+1. Insert your USB stick (select 'Do nothing' if prompted by Ubuntu).
+1. Use the top left icon to open the dash and search for *Startup Disk Creator*.
+1. Select **Startup Disk Creator** from the results to launch the application.
 
-Click '**Other**' to choose the downloaded ISO file if it isn’t found automatically, select the file and click '**Open**'
+![screenshot](https://assets.ubuntu.com/v1/e069dea2-ubuntu-startup-launch.png)
 
-![screenshot](https://assets.ubuntu.com/v1/a2cf98cc-usb-1604-select-iso.png?w=550)
+## ISO and USB selection
+Duration: 0:02
 
-## Select the ISO
-Duration: 0:10
+When launched, Startup Disk Creator will automatically find your ISO files, as well as any attached USB storage it can write to.
```

Comment:
It only finds ones in the 'Downloads' folder. I don't know if this is worth mentioning